<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nesting</string>
  </property>
  <widget class="QPushButton" name="BrowseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Browse</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GeneticAlgorithmGroup">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Genetic Algorithm</string>
   </property>
   <widget class="QSpinBox" name="PopulationSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PopulationLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Population</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="IterationSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IterationLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MutationSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MutationLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutation Rate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpawnSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SpawnLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Spawn Rate</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="NestingGroup">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Nesting Process</string>
   </property>
   <widget class="QLabel" name="MagetLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnet</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MagnetCombo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OpenLoopAngle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Loop Action</string>
    </property>
   </widget>
   <widget class="QComboBox" name="OpenLoopCombo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InnerShapeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow Inner Shapes</string>
    </property>
   </widget>
   <widget class="QComboBox" name="InnerShapeCombo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LinearStepLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear Step Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="AngularStepLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Angular Step Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxWidthLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Material Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxHeightLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Material Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="KerfLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kerf Offset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MaxHeightSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="KerfSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="AngularStepSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="MaxWidthSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="LinearStepCombo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ResolutionSpinner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="KerfLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rasterization Resolution</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GeometryGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Geometry Set</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="GeometryListLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RunButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
